cmd/agent/main.go

```diff
@@ -485,3 +483,18 @@ func reconcileKubelet(ctx context.Context) (bool, error) {
 	}
 	return false, nil
 }
+
+func setLogrus() {
+	logrus.SetOutput(colorable.NewColorableStdout())
+
+	if os.Getenv("CATTLE_TRACE") == "true" || os.Getenv("RANCHER_TRACE") == "true" {
+		logrus.SetLevel(logrus.TraceLevel)
+		return
+	}
+
+	if os.Getenv("CATTLE_DEBUG") == "true" || os.Getenv("RANCHER_DEBUG") == "true" {
+		logrus.SetLevel(logrus.DebugLevel)
+	}
+
+	return
```

Comment:
This return statement isnt necessary

Reply:
Yes. We used to always use explicit returns.

I'll remove it.